Answers.md

```diff
@@ -0,0 +1,14 @@
+Question 1:
+    The class that has the most specificity weight is box3.
```

Comment:
Exactly, when all of the elements have the same weight, it goes to the last one!